### PR TITLE
Various edits

### LINKED ISSUE #1: Catch up with the latest updates on the eForms award criterion model (BG-38)

### PR DESCRIPTION
**Pull request by jpmckinney**
closes #1 

Readme

* Removed mapping guidance (should only be in OCDS for EU profile)
* Put EU-specific content in "Legal context" section
* Added example
* Added links

Codelists

* Added new codelists from latest eForms
* Remove empty Deprecated column
* Renamed criterionType.csv to awardCriterionType.csv and simplified descriptions compared to awardCriteria.csv

Schema

* Put Tender at top of patch
* Rename `awardWeightedCriteriaPerLot` to `awardCriteriaBreakdown` (to match budget breakdown extension)
* Remove `Weighted` word from definition names, as criteria are not necessarily weighted (e.g. order of importance is not a weight per 2014/24/EU)
* Rename `lotID` to `relatedLots` (to match lots extension)
* Remove repetitive prefixes: `awardCriteriaComplicated` -> `weightingDescription`, `awardCriteriaOrderJustification` -> `orderJustification`, `criterionType` -> `type`, `criterionName` -> `name`, `criterionDescription` -> `description`, `criterionNumber` -> `numbers`
* Add new fields from latest eForms
* Tidy titles and descriptions, including removing EU-specific references from descriptions
* Consistently use the property order: `title`, `description`, `type`
* Add `wholeListMerge` properties

I also removed the option to identify groups of lots. The lots extension doesn't require lot IDs and lot group IDs to be distinct. So, we'd need two properties to identify lots and lot groups, which becomes complicated. Instead, we can refer to the lots directly.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,53 +1,56 @@
...
+```json
+{
+  "tender": {
+    "awardCriteriaBreakdown": [
+      {
+        "relatedLots": [
+          "1",
+          "2"
+        ],
+        "awardCriteria": [
```
Comment (ColinMaudry) [inline README.md:23]: Could you add an example with `fixed`? I don't understand how it's used.

Comment (jpmckinney) [reply README.md:23]: Yes. Otherwise, the changes look good?

Comment (jpmckinney) [reply README.md:23]: I added an example, based on https://www.felp.ac.uk/content/most-economically-advantageous-tender-meat and https://github.com/eprocurementontology/eprocurementontology/issues/157

**Comment by jpmckinney** [comment]
As discussed, I moved `awardCriteriaBreakdown` from Tender to `awardCriteria` on Lot. Renamed the deep `awardCriteria` field to just `criteria`, since it is already in context of award criteria.

I also renamed `orderJustification` to `orderRationale` for consistency.

**Pull request merged**